### ISSUE TITLE
Filter the posts by the content author type (person or company)

### ISSUE THREAD

**Issue by ladaflac** [njelich/LinkOff#6]
I'd like to filter the posts by the content author, eg.- a person vs. a company page. 

I'm not sure if there is other way to differentiate between those two groups except checking the url of the poster, where the urls of the companies start with linkedin.com/company/ and personal profiles start with linkedin.com/in/. 

In case of the shared posts, where a person is sharing a company post or a company sharing a person's post, the filter should apply only to the poster that I am following and not the author of the shared post.

**Comment by njelich** [comment]
Yes, there is a field with a href to the company page on LinkedIn in company posts, while the same field contains a personal profile link for posts by people.

`<a href="https://www.linkedin.com/company/companyname"></a>`

versus

`<a href="https://www.linkedin.com/in/profile"></a>`
